Source/WebCore/platform/mediastream/gstreamer/RealtimeOutgoingMediaSourceGStreamer.cpp

```diff
@@ -433,7 +433,7 @@ void RealtimeOutgoingMediaSourceGStreamer::setParameters(GUniquePtr<GstStructure
         if (!rid)
             continue;
 
-        auto packetizer = getPacketizerForRid(rid.toString());
+        auto packetizer = getPacketizerForRid(String(rid.span()));
```

Comment:
Why do we have to explicitly write `String()` here? Doesn't it compile without it?